packages/host/app/resources/import.ts

```diff
@@ -1,40 +1,40 @@
-import { getOwner } from '@ember/owner';
+import { service } from '@ember/service';
 import { tracked } from '@glimmer/tracking';
 
 import { task } from 'ember-concurrency';
 import { Resource } from 'ember-resources';
 
 import { logger } from '@cardstack/runtime-common';
-import { Loader } from '@cardstack/runtime-common/loader';
 
 import type LoaderService from '../services/loader-service';
 
 interface Args {
-  named: { url: string; loader: Loader };
+  named: { url: string };
 }
 
 const log = logger('resource:import');
 
 export class ImportResource extends Resource<Args> {
   @tracked module: object | undefined;
   @tracked error: { type: 'runtime' | 'compile'; message: string } | undefined;
+  @service declare loaderService: LoaderService;
```

Comment:
same comment as above--let's try to limit our use of the loader service when possible